backend/index.js

```diff
@@ -1,13 +1,107 @@
 import express from 'express';
+import dotenv from "dotenv";
+// import passport from "passport";
+import axios from "axios";
+import cors from "cors";
+import bodyParser from "body-parser";
+import jwt from "jsonwebtoken";
+
+axios.defaults.headers.post['Accept'] = 'application/json';
+
+dotenv.config();
+
+
 
 const app = express();
 
+app.use(bodyParser.json());
+
+app.use(cors());
```

Comment:
Why do we need `cors`? I believe all requests will come from the same domain as the page is hosted from, so this should not be necessary.